server/src/main/java/com/objectcomputing/checkins/services/member_skill/skillsreport/SkillLevelDTO.java

```diff
@@ -0,0 +1,35 @@
+package com.objectcomputing.checkins.services.member_skill.skillsreport;
+
+import io.micronaut.core.annotation.Introspected;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import javax.annotation.Nullable;
+import java.util.UUID;
+
+@Introspected
+public class SkillLevelDTO {
+    @NotNull
+    @Schema(required = true, description = "UUID of the skill")
+    private UUID id;
+
+    @Nullable
+    @Schema(description = "Level of the skill")
+    private String level;
```

Comment:
should we make this an `enum` instead of `string`?

Reply:
I agree that `enum` should be used. This is just so that it's compatible with the existing definition of skill level in member_skill service, which uses `string`. If we decide to change to `string`, I can do that in this PR.

Reply:
I would like to change the database and the related entities/dtos to use an Integer instead of a string.